src/org/mozilla/javascript/GetterSlot.java

```diff
@@ -0,0 +1,194 @@
+package org.mozilla.javascript;
+
+/**
+ * This is a specialization of Slot to store various types of values that are retrieved dynamically
+ * using Java and JavaScript functions. Unlike LambdaSlot, the fact that these values are accessed
+ * and mutated by functions is visible via the slot's property descriptor.
+ */
+public class GetterSlot extends Slot {
```

Comment:
Maybe AccessorSlot?

Reply:
Yeah, we had that name before but AccessorSlot is better.